osquery/tables/applications/browser_utils.cpp

```diff
@@ -31,6 +31,8 @@ const std::map<std::string, std::string> kExtensionKeys = {
     {"author", "author"},
     {"background.persistent", "persistent"}};
 
+const std::string kExtensionPermissionKey = "permissions";
```

Comment:
It can be hidden in the anonymous namespace

Reply:
I've added the anonymous namespace around the global variables